tests/backup/backup_sse_test.go

```diff
@@ -0,0 +1,412 @@
+				}
+				wg.Wait()
+			})
+			Step("Create restore with replace policy set to retain", func() {
+				log.InfoD("Create restore with replace policy set to retain")
+				restoreName := fmt.Sprintf("restore-with-replace-%s-%v", RestoreNamePrefix, time.Now().Unix())
+				err = CreateRestoreWithReplacePolicy(restoreName, backupsWithSse[0], make(map[string]string), SourceClusterName, orgID, ctx, make(map[string]string), 2)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore [%s]", restoreName))
+				restoreList = append(restoreList, restoreName)
+			})
+			Step("Create new storage class on source cluster for storage class mapping for restore", func() {
+				log.InfoD("Create new storage class on source cluster for storage class mapping for restore")
+				scCount = 3
+				for i := 0; i < scCount; i++ {
+					scName := fmt.Sprintf("replica-sc-%d-%v", time.Now().Unix(), i)
+					params["repl"] = "2"
+					v1obj := metaV1.ObjectMeta{
+						Name: scName,
+					}
+					reclaimPolicyDelete := v1.PersistentVolumeReclaimDelete
+					bindMode := storageApi.VolumeBindingImmediate
+					scObj := storageApi.StorageClass{
+						ObjectMeta:        v1obj,
+						Provisioner:       k8s.CsiProvisioner,
+						Parameters:        params,
+						ReclaimPolicy:     &reclaimPolicyDelete,
+						VolumeBindingMode: &bindMode,
+					}
+
+					_, err := k8sStorage.CreateStorageClass(&scObj)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creating new storage class %v on source cluster %s", scName, SourceClusterName))
+					scNames = append(scNames, scName)
+				}
+			})
+			Step("Multiple restore for same backup in different storage class at the same time", func() {
+				log.InfoD(fmt.Sprintf("Multiple restore for same backup into %d different storage class at the same time", scCount))
+				ctx, err := backup.GetAdminCtxFromSecret()
+				log.FailOnError(err, "Fetching px-central-admin ctx")
+				pvcs, err := core.Instance().GetPersistentVolumeClaims(bkpNamespaces[2], make(map[string]string))
+				singlePvc := pvcs.Items[0]
+				sourceScName, err = core.Instance().GetStorageClassForPVC(&singlePvc)
+				var wg sync.WaitGroup
+				for i := 0; i < scCount; i++ {
+					storageClassMapping[sourceScName.Name] = scNames[i]
+					time.Sleep(2)
+					namespaceMap[bkpNamespaces[2]] = fmt.Sprintf("new-namespace-%v", time.Now().Unix())
+					restoreName := fmt.Sprintf("restore-new-storage-class-%s-%s", scNames[i], RestoreNamePrefix)
+					restoreList = append(restoreList, restoreName)
+					wg.Add(1)
+					go func(scName string) {
+						defer GinkgoRecover()
+						defer wg.Done()
+						err = CreateRestore(restoreName, backupsWithSse[0], namespaceMap, SourceClusterName, orgID, ctx, storageClassMapping)
+						dash.VerifyFatal(err, nil, fmt.Sprintf("Restoring backup %v using storage class %v", backupName, scName))
+					}(scNames[i])
+				}
+				wg.Wait()
+			})
+			//Step("Update backup location backupLocationUidWithoutSse to sse true", func() {
```

Comment:
why is this commented ?

Reply:
It is due to product issue ,Once this update issue is fixed i will enable it
 https://portworx.atlassian.net/browse/PB-4687

Reply:
Please add a `TODO` with the bug ID